src/index.html

```diff
@@ -5,8 +5,288 @@
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__prices">
+              <p class="card__text">Price:</p>
+              <p class="card__price">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="card__link">
+              BUY
+            </a>
+          </div>
+        </div>
+        <div class="card">
+          <img class="card__image"
+           src="./images/imac.jpeg"
+           alt="APPLE A1419 iMac 27"
+          >
+          <div class="card__content">
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <h3 class="card__subtitle">Product code: 195434</h3>
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__prices">
+              <p class="card__text">Price:</p>
+              <p class="card__price">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="card__link">
+              BUY
+            </a>
+          </div>
+        </div>
+        <div class="card">
+          <img class="card__image"
+           src="./images/imac.jpeg"
+           alt="APPLE A1419 iMac 27"
+          >
+          <div class="card__content">
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <h3 class="card__subtitle">Product code: 195434</h3>
+            <div class="card__stars">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__reviews">Reviews: 5</p>
+            </div>
+            <div class="card__prices">
+              <p class="card__text">Price:</p>
+              <p class="card__price">$2,199</p>
+            </div>
+            <a
+              href="#"
+              class="card__link">
+              BUY
+            </a>
+          </div>
+        </div>
+        <div class="card">
+          <img class="card__image"
+           src="./images/imac.jpeg"
+           alt="APPLE A1419 iMac 27"
+          >
+          <div class="card__content">
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <h3 class="card__subtitle">Product code: 195434</h3>
+            <div class="card__stars">
+              <div class="stars stars--4">
```

Comment:
```suggestion
        <div class="card">
          <img
           class="card__image"
           src="./images/imac.jpeg"
           alt="APPLE A1419 iMac 27"
          >

          <div class="card__content">
            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>

            <h3 class="card__subtitle">Product code: 195434</h3>

            <div class="card__stars">
              <div class="stars stars--4">
```
to increase readability, add empty lines between _sibling_ tags (no need empty lines between parent and child tags)